packages/react/src/components/layout/Drawer/index.tsx

```diff
@@ -26,16 +26,12 @@ const Footer = styled(FlexBox)`
   padding: ${(p) => p.theme.space[8]}px ${(p) => p.theme.space[12]}px;
 `;
 
-const Wrapper = styled.div<{
+const Wrapper = styled(FlexBox)<{
   big?: boolean;
-  width?: number;
-  height?: number;
-  backgroundColor?: string;
 }>`
   height: 100%;
   width: ${(p) =>
     p.big ? p.theme.sizes.drawer.side.big.width : p.theme.sizes.drawer.side.small.width}px;
-  background-color: ${(p) => p.backgroundColor ?? p.theme.colors.neutral.c00};
   display: flex;
```

Comment:
we don't need this line anymore I think 🧹

Reply:
Good catch! Updated it.